From 7ce97eefb05afd1d155a0fd3fe816b0939055893 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Mon, 24 Aug 2020 16:45:05 +0200
Subject: [PATCH] COMP: Fix linking issue with VolumeRendering

- This adds qSlicerBaseQTGUI as link library for SubjectHierarchyPlugins
in VolumeRendering
---
 .../VolumeRendering/SubjectHierarchyPlugins/CMakeLists.txt       | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/Loadable/VolumeRendering/SubjectHierarchyPlugins/CMakeLists.txt b/Modules/Loadable/VolumeRendering/SubjectHierarchyPlugins/CMakeLists.txt
index 757cfb1487..5448d67057 100644
--- a/Modules/Loadable/VolumeRendering/SubjectHierarchyPlugins/CMakeLists.txt
+++ b/Modules/Loadable/VolumeRendering/SubjectHierarchyPlugins/CMakeLists.txt
@@ -37,6 +37,7 @@ set(${KIT}_TARGET_LIBRARIES
   vtkSlicer${MODULE_NAME}ModuleLogic
   qSlicerSubjectHierarchyModuleWidgets
   vtkSlicerSubjectHierarchyModuleLogic
+  qSlicerBaseQTGUI
   qMRMLWidgets
   MRMLLogic
   MRMLCore
-- 
2.26.2

